plugin/src/main/resources/org/jenkinsci/plugins/workflow/cps/config/CPSConfiguration/help-hideSandbox.html

```diff
@@ -0,0 +1,6 @@
+<div>
+    <p>Controls whether the "Use Groovy Sandbox" is shown in pipeline jobs configuration page to users without Overall/Administer permission.</p>
+    <p>This can be used to get a better UX in highly secured environments where all pipelines are required to run in the sandbox (ie. running arbitrary code is never approved)</p>
+    <p>Note that this does not prevent users to configure and run pipelines with sandbox disabled if they create or update jobs by other means (like CLI or HTTP API).
```

Comment:
So for now we are not touching `readResolve`, deliberately?

Reply:
Yes. I don't see how to differentiate from a regular call during start up and a call from a CLI/HTTP API call.

Reply:
Deserialization during startup would run as `SYSTEM`, which of course has `ADMINISTER`; deserialization during a CLI/HTTP call would run as some user authentication.

Reply:
IMO, we should leave this as is and revisit it separately if/when needed/requested.